### PR TITLE
Adding blog post for VS2022 support in O3DE

### PR DESCRIPTION
**Pull request by lumberyard-employee-dm**
Signed-off-by: lumberyard-employee-dm <56135373+lumberyard-employee-dm@users.noreply.github.com>

## Change summary

Introduces information about the VS2022 support now available in O3DE as of the 22.10.0 release.
The blog post also provides some tricks about using VS2022 with O3DE and how to mitigate memory issues when compiling.

### Submission Checklist:

* [ ] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [ ] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [ ] **Consistency** - Does the content consistently follow the [Style Guide](https://o3de.org/docs/contributing/to-docs/style-guide/quick-reference)?
* [ ] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: content/blog/posts/visual-studio-2022-update.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+full_img: ""
+---
+With the 22.10.0 release of the **Open 3D Engine(O3DE)** Visual Studio 2022(VS2022) is now fully supported within the O3DE.
+
+## Background
+
+When we say that **O3DE** "fully supports" Visual Studio 2022, we are referring to building O3DE projects using Visual Studio 2022 [v143 toolset](https://learn.microsoft.com/en-us/cpp/build/how-to-modify-the-target-framework-and-platform-toolset?view=msvc-170#platform-toolset).  
+![Visual Studio 2022 v143 toolset](/images/blog/vs2022/vs2022-installer-v143-toolset.png)
+
+Because O3DE uses [CMake](https://www.o3de.org/docs/user-guide/build/configure-and-build/) as its build system generator, it has been possible since the release of [CMake 3.21](https://cmake.org/cmake/help/v3.21/release/3.21.html#generators) to configure and *attempt* to build using the Visual Studio 2022.  By *attempt*, we mean that building O3DE using Visual Studio 2022 could occur, but would have most likely failed due to stricter compiler warnings which were triggering using the newer v143 toolset.
```
Comment (aFinchy) [inline content/blog/posts/visual-studio-2022-update.md:16]: [CMake](https://www.o3de.org/docs/user-guide/build/configure-and-build/)
Change to
[CMake](/docs/user-guide/build/configure-and-build/)

File: content/blog/posts/visual-studio-2022-update.md [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+author: lumberyard-employee-dm
+blog_img: "/images/blog/announcement_thumbnail.jpg"
+full_img: ""
+---
+With the 22.10.0 release of the **Open 3D Engine(O3DE)** Visual Studio 2022(VS2022) is now fully supported within the O3DE.
+
+## Background
+
+When we say that **O3DE** "fully supports" Visual Studio 2022, we are referring to building O3DE projects using Visual Studio 2022 [v143 toolset](https://learn.microsoft.com/en-us/cpp/build/how-to-modify-the-target-framework-and-platform-toolset?view=msvc-170#platform-toolset).  
+![Visual Studio 2022 v143 toolset](/images/blog/vs2022/vs2022-installer-v143-toolset.png)
```
Comment (aFinchy) [inline content/blog/posts/visual-studio-2022-update.md:14]: `Visual Studio 2022 v143 toolset`
Change to
`{{< image-width src="/images/blog/vs2022/vs2022-installer-v143-toolset.png" width="100%" alt="Visual Studio 2022 v143 toolset" >}}`

File: content/blog/posts/visual-studio-2022-update.md [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+Furthermore all O3DE build warnings and errors using the the v143 toolset were fixed and future VS2022 compile failures are protected against via the Automated Review pipeline validating each Pull Request that goes into O3DE repo [development](https://github.com/o3de/o3de) branch.  
+
+
+### Project Manager Updates
+The 22.10.0 release also included a feature update to the [Project Manager](https://www.o3de.org/docs/user-guide/project-config/project-manager/#projects) application. The requirement building against the Visual Studio 2019 application has been removed.  
+The Project Manager application will use the build toolset of latest version of Visual Studio installed on the user's machine for configuring and building O3DE Projects.  If the user only has Visual Studio 2019 installed with the v142 toolset, that will be used for compiling O3DE. If the user only has Visual Studio 2022 installed with the v143 toolset that will be used for compilation. Finally if the user has both Visual Studio 2019 and 2022 installed then Visual Studio 2022 v143 toolset will be used.
+![Project Manager Project Options](/images/blog/vs2022/project-manager-build-options.png)
```
Comment (aFinchy) [inline content/blog/posts/visual-studio-2022-update.md:30]: `Project Manager Project Options`
Change to
`{{< image-width src="/images/blog/vs2022/project-manager-build-options.png" width="100%" alt="Project Manager Project Options" >}}`

File: content/blog/posts/visual-studio-2022-update.md [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+
+When building from the command line on a machine with 64 logical cores and 64 GiB of ram, the parameters of `/maxcpucount` and `/p:CL_MPCount` can be supplied to CMake build wrapper command(`cmake --build`) to control the number of cl.exe processes that get launched.
+The following command restricts msbuild to build no more than 8 Visual Studio C++ Projects(.vcxproj) in parallel at a given time and no more than 16 `cl.exe` process per .vcxproj at once.
+```
+cmake --build build/windows --target Editor --config profile -- /maxcpucount:8 /p:CL_MPCount=16
+```
+
+The equivalent options in the Visual Studio 2022 IDE can be set via following IDE options
+* Setting the maximum number of parallel project builds value in the `Tools -> Options -> Projects and Solutions -> Build and Run` menu.  
+![Build and Run](/images/blog/vs2022/vs2022-parallel-project-builds-8.png)
```
Comment (aFinchy) [inline content/blog/posts/visual-studio-2022-update.md:54]: `Build and Run`
Change to
`{{< image-width src="/images/blog/vs2022/vs2022-parallel-project-builds-8.png" width="100%" alt="Build and Run" >}}`

File: content/blog/posts/visual-studio-2022-update.md [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
+* Setting the maximum concurrent C++ compilations option in the `Tools -> Options -> Projects and Solutions -> VC++ Project Settings` menu.  
+![VC++ Project Settings](/images/blog/vs2022/vs2022-concurrent-compiliations-16.png)
```
Comment (aFinchy) [inline content/blog/posts/visual-studio-2022-update.md:56]: `VC++ Project Settings`
Change to
`{{< image-width src="/images/blog/vs2022/vs2022-concurrent-compiliations-16.png" width="100%" alt="VC++ Project Settings" >}}`

File: content/blog/posts/visual-studio-2022-update.md [hunk 6]
```diff
@@ -0,0 +1,277 @@
...
+To use the the latest v142 toolset which is 14.29 at this time, the following value for the `-T` option should be supplied:
+```
+-Tv142
+```
+
+
+#### Configuring the CMake Gui to use the v142 toolset using VS 2022
+
+Using the CMake GUI to build against the VS2019 v142 toolset is as simple as specifying v142 to the toolset parameter
+![CMake GUI Visual Studio Toolset](/images/blog/vs2022/cmake-gui-14-29-toolset.png)
```
Comment (aFinchy) [inline content/blog/posts/visual-studio-2022-update.md:86]: `CMake GUI Visual Studio Toolset`
Change to
`{{< image-width src="/images/blog/vs2022/cmake-gui-14-29-toolset.png" width="100%" alt="CMake GUI Visual Studio Toolset" >}}`

File: content/blog/posts/visual-studio-2022-update.md [hunk 7]
```diff
@@ -0,0 +1,277 @@
...
+-- The C compiler identification is MSVC 19.33.31630.0
+-- The CXX compiler identification is MSVC 19.33.31630.0
+```
+
+### Using O3DE CMake Presets (CMake 3.23+)
+O3DE provides a [CMakePresets.json](https://github.com/o3de/o3de/blob/main/CMakePresets.json) in the repo root, which leverages the [CMakePresets](https://cmake.org/cmake/help/v3.23/manual/cmake-presets.7.html) feature which is way to shared configuration, build and test settings with other users.  
+{{< note >}} This feature requires using CMake 3.23 or above installed on the user's machine {{< /note >}}
+
+The list of available CMake presets for the current host platform can be seen in the CMake GUI or the via the `cmake --list-presets` command
+![O3DE CMake Presets](/images/blog/vs2022/cmake-presets-gui.png)
```
Comment (aFinchy) [inline content/blog/posts/visual-studio-2022-update.md:116]: `O3DE CMake Presets`
Change to
`{{< image-width src="/images/blog/vs2022/cmake-presets-gui.png" width="100%" alt="O3DE CMake Presets" >}}`

File: content/blog/posts/visual-studio-2022-update.md [hunk 8]
```diff
@@ -0,0 +1,277 @@
...
+
+## What's new
+
+### Build Server Updates
+Between the time of the 22.05.0(May 2022) release the 22.10.0(October 2022) release the Jenkins build servers have been updated with Visual Studio 2022 17.3 Build Tools and the latest v143 toolset. 
+Furthermore all O3DE build warnings and errors using the the v143 toolset were fixed and future VS2022 compile failures are protected against via the Automated Review pipeline validating each Pull Request that goes into O3DE repo [development](https://github.com/o3de/o3de) branch.  
+
+
+### Project Manager Updates
+The 22.10.0 release also included a feature update to the [Project Manager](https://www.o3de.org/docs/user-guide/project-config/project-manager/#projects) application. The requirement building against the Visual Studio 2019 application has been removed.
```
Comment (aFinchy) [inline content/blog/posts/visual-studio-2022-update.md:28]: [Project Manager](https://www.o3de.org/docs/user-guide/project-config/project-manager/#projects)
Change to 
[Project Manager](/docs/user-guide/project-config/project-manager/#projects)

**Review by aFinchy — CHANGES_REQUESTED** [review]
Left a few comments on various changes to be made. Most are more formatting related issues. 

Other than that, great blog.

File: content/blog/posts/visual-studio-2022-update.md [hunk 9]
```diff
@@ -0,0 +1,277 @@
...
+Because O3DE uses [CMake](https://www.o3de.org/docs/user-guide/build/configure-and-build/) as its build system generator, it has been possible since the release of [CMake 3.21](https://cmake.org/cmake/help/v3.21/release/3.21.html#generators) to configure and *attempt* to build using the Visual Studio 2022.  By *attempt*, we mean that building O3DE using Visual Studio 2022 could occur, but would have most likely failed due to stricter compiler warnings which were triggering using the newer v143 toolset.  
+The backend build server infrastructure for validating new commits to O3DE via the Automated Review(AR) pipeline wasn't setup to build against the v143 toolset.  
+Before the 22.10.0 release, continuous integration builds on Windows only installed the Visual Studio 2019(VS2019) Build Tools. The [v142 toolset](https://learn.microsoft.com/en-us/cpp/build/how-to-modify-the-target-framework-and-platform-toolset?view=msvc-170#platform-toolset) was the only Microsoft Visual C++(MSVC) toolset validated at the time and therefore build breakages of O3DE building against the Visual Studio 2022 v143 toolset were common and took days to discover.
+
+## What's new
+
+### Build Server Updates
```
Comment (amzn-changml) [inline content/blog/posts/visual-studio-2022-update.md:22]: Maybe we should call these "CI Nodes" or " CI Runner" instead of "Server"? It might make it clear to folks that this is specific to the CI/AR process?

**Review by amzn-changml — COMMENTED** [review]
Aside from the formatting/link comments from @aFinchy, this looks good from a technical standpoint. Some minor comments that are non-blocking.

File: content/blog/posts/visual-studio-2022-update.md (hunk 9 above)
Comment (lumberyard-employee-dm) [reply content/blog/posts/visual-studio-2022-update.md:22]: I'll update it to use the terminology of CI Nodes for the Continuous Integration

File: content/blog/posts/visual-studio-2022-update.md (hunk 8 above)
Comment (lumberyard-employee-dm) [reply content/blog/posts/visual-studio-2022-update.md]: I changed the link to `/docs/user-guide/project-config/project-manager/#projects`

**Review by amzn-changml — APPROVED** [review]

File: content/blog/posts/visual-studio-2022-update.md [hunk 10]
```diff
@@ -0,0 +1,277 @@
+---
+title: "O3DE Visual Studio 2022 Support!"
+date: 2022-11-09
+slug: o3de-vs2022-initial-support
+author: lumberyard-employee-dm
+blog_img: "/images/blog/announcement_thumbnail.jpg"
+full_img: ""
+---
+With the 22.10.0 release of the **Open 3D Engine(O3DE)** Visual Studio 2022(VS2022) is now fully supported within the O3DE.
```
Comment (hultonha) [inline content/blog/posts/visual-studio-2022-update.md:9]: uber nit: I think there should be a space after `Engine` before `(O3DE)`

File: content/blog/posts/visual-studio-2022-update.md [hunk 11]
```diff
@@ -0,0 +1,277 @@
...
+
+## Background
+
+When we say that **O3DE** "fully supports" Visual Studio 2022, we are referring to building O3DE projects using Visual Studio 2022 [v143 toolset](https://learn.microsoft.com/en-us/cpp/build/how-to-modify-the-target-framework-and-platform-toolset?view=msvc-170#platform-toolset).  
+{{< image-width src="/images/blog/vs2022/vs2022-installer-v143-toolset.png" width="100%" alt="Visual Studio 2022 v143 toolset" >}}
+
+Because O3DE uses [CMake](/docs/user-guide/build/configure-and-build/) as its build system generator, it has been possible since the release of [CMake 3.21](https://cmake.org/cmake/help/v3.21/release/3.21.html#generators) to configure and *attempt* to build using the Visual Studio 2022.  By *attempt*, we mean that building O3DE using Visual Studio 2022 could occur, but would have most likely failed due to stricter compiler warnings which were triggering using the newer v143 toolset.
```
Comment (hultonha) [inline content/blog/posts/visual-studio-2022-update.md:16]: change: `build using the Visual Studio 2022` -> `build using Visual Studio 2022`

File: content/blog/posts/visual-studio-2022-update.md [hunk 12]
```diff
@@ -0,0 +1,277 @@
...
+
+## Background
+
+When we say that **O3DE** "fully supports" Visual Studio 2022, we are referring to building O3DE projects using Visual Studio 2022 [v143 toolset](https://learn.microsoft.com/en-us/cpp/build/how-to-modify-the-target-framework-and-platform-toolset?view=msvc-170#platform-toolset).  
+{{< image-width src="/images/blog/vs2022/vs2022-installer-v143-toolset.png" width="100%" alt="Visual Studio 2022 v143 toolset" >}}
+
+Because O3DE uses [CMake](/docs/user-guide/build/configure-and-build/) as its build system generator, it has been possible since the release of [CMake 3.21](https://cmake.org/cmake/help/v3.21/release/3.21.html#generators) to configure and *attempt* to build using the Visual Studio 2022.  By *attempt*, we mean that building O3DE using Visual Studio 2022 could occur, but would have most likely failed due to stricter compiler warnings which were triggering using the newer v143 toolset.  
+The backend continuous integration(CI) build infrastructure for validating new commits to O3DE via the Automated Review(AR) pipeline wasn't setup to build against the v143 toolset.
```
Comment (hultonha) [inline content/blog/posts/visual-studio-2022-update.md:17]: nit: Space after `integration` before `CI`

File: content/blog/posts/visual-studio-2022-update.md [hunk 13]
```diff
@@ -0,0 +1,277 @@
...
+
+Because O3DE uses [CMake](/docs/user-guide/build/configure-and-build/) as its build system generator, it has been possible since the release of [CMake 3.21](https://cmake.org/cmake/help/v3.21/release/3.21.html#generators) to configure and *attempt* to build using the Visual Studio 2022.  By *attempt*, we mean that building O3DE using Visual Studio 2022 could occur, but would have most likely failed due to stricter compiler warnings which were triggering using the newer v143 toolset.  
+The backend continuous integration(CI) build infrastructure for validating new commits to O3DE via the Automated Review(AR) pipeline wasn't setup to build against the v143 toolset.  
+Before the 22.10.0 release, continuous integration builds on Windows only installed the Visual Studio 2019(VS2019) Build Tools. The [v142 toolset](https://learn.microsoft.com/en-us/cpp/build/how-to-modify-the-target-framework-and-platform-toolset?view=msvc-170#platform-toolset) was the only Microsoft Visual C++(MSVC) toolset validated at the time and therefore build breakages of O3DE building against the Visual Studio 2022 v143 toolset were common and took days to discover.
```
Comment (hultonha) [inline content/blog/posts/visual-studio-2022-update.md:18]: update: `builds on Windows only installed the Visual Studio...` -> `builds on Windows only installed with Visual Studio...`

File: content/blog/posts/visual-studio-2022-update.md [hunk 14]
```diff
@@ -0,0 +1,277 @@
...
+
+## What's new
+
+### CI Nodes Updates
+Between the time of the 22.05.0(May 2022) release the 22.10.0(October 2022) release the Jenkins CI Nodes have been updated with Visual Studio 2022 17.3 Build Tools and the latest v143 toolset.
```
Comment (hultonha) [inline content/blog/posts/visual-studio-2022-update.md:23]: update: `the time of the 22.05.0(May 2022) release the 22.10.0(October 2022) the...` -> `the time of the 22.05.0 (May 2022) release and the 22.10.0 (October 2022) release, the...`

File: content/blog/posts/visual-studio-2022-update.md [hunk 15]
```diff
@@ -0,0 +1,277 @@
...
+The backend continuous integration(CI) build infrastructure for validating new commits to O3DE via the Automated Review(AR) pipeline wasn't setup to build against the v143 toolset.  
+Before the 22.10.0 release, continuous integration builds on Windows only installed the Visual Studio 2019(VS2019) Build Tools. The [v142 toolset](https://learn.microsoft.com/en-us/cpp/build/how-to-modify-the-target-framework-and-platform-toolset?view=msvc-170#platform-toolset) was the only Microsoft Visual C++(MSVC) toolset validated at the time and therefore build breakages of O3DE building against the Visual Studio 2022 v143 toolset were common and took days to discover.
+
+## What's new
+
+### CI Nodes Updates
+Between the time of the 22.05.0(May 2022) release the 22.10.0(October 2022) release the Jenkins CI Nodes have been updated with Visual Studio 2022 17.3 Build Tools and the latest v143 toolset. 
+Furthermore all O3DE build warnings and errors using the the v143 toolset were fixed and future VS2022 compile failures are protected against via the Automated Review pipeline validating each Pull Request that goes into O3DE repo [development](https://github.com/o3de/o3de) branch.
```
Comment (hultonha) [inline content/blog/posts/visual-studio-2022-update.md:24]: nit: `O3DE repo` -> `the O3DE repo` (here the 'the' makes more sense as we're referring to the repo, not just O3DE

File: content/blog/posts/visual-studio-2022-update.md [hunk 16]
```diff
@@ -0,0 +1,277 @@
...
+Furthermore all O3DE build warnings and errors using the the v143 toolset were fixed and future VS2022 compile failures are protected against via the Automated Review pipeline validating each Pull Request that goes into O3DE repo [development](https://github.com/o3de/o3de) branch.  
+
+
+### Project Manager Updates
+The 22.10.0 release also included a feature update to the [Project Manager](/docs/user-guide/project-config/project-manager/#projects) application. The requirement of Project Manager only being able to build using the Visual Studio 2019 build tools has been removed.  
+The Project Manager application will use the build toolset of latest version of Visual Studio installed on the user's machine for configuring and building O3DE Projects.  If the user only has Visual Studio 2019 installed with the v142 toolset, that will be used for compiling O3DE. If the user only has Visual Studio 2022 installed with the v143 toolset that will be used for compilation. Finally if the user has both Visual Studio 2019 and 2022 installed then Visual Studio 2022 v143 toolset will be used.
```
Comment (hultonha) [inline content/blog/posts/visual-studio-2022-update.md:29]: update: `toolset of latest version` -> `toolset of the latest version`

File: content/blog/posts/visual-studio-2022-update.md [hunk 17]
```diff
@@ -0,0 +1,277 @@
...
+
+## What's new
+
+### CI Nodes Updates
+Between the time of the 22.05.0(May 2022) release the 22.10.0(October 2022) release the Jenkins CI Nodes have been updated with Visual Studio 2022 17.3 Build Tools and the latest v143 toolset. 
+Furthermore all O3DE build warnings and errors using the the v143 toolset were fixed and future VS2022 compile failures are protected against via the Automated Review pipeline validating each Pull Request that goes into O3DE repo [development](https://github.com/o3de/o3de) branch.  
+
+
+### Project Manager Updates
+The 22.10.0 release also included a feature update to the [Project Manager](/docs/user-guide/project-config/project-manager/#projects) application. The requirement of Project Manager only being able to build using the Visual Studio 2019 build tools has been removed.
```
Comment (hultonha) [inline content/blog/posts/visual-studio-2022-update.md:28]: nit: Adding a blank line after # headings would be good to be consistent (it's also enforced by Markdown linters)

File: content/blog/posts/visual-studio-2022-update.md [hunk 18]
```diff
@@ -0,0 +1,277 @@
...
+The 22.10.0 release also included a feature update to the [Project Manager](/docs/user-guide/project-config/project-manager/#projects) application. The requirement of Project Manager only being able to build using the Visual Studio 2019 build tools has been removed.  
+The Project Manager application will use the build toolset of latest version of Visual Studio installed on the user's machine for configuring and building O3DE Projects.  If the user only has Visual Studio 2019 installed with the v142 toolset, that will be used for compiling O3DE. If the user only has Visual Studio 2022 installed with the v143 toolset that will be used for compilation. Finally if the user has both Visual Studio 2019 and 2022 installed then Visual Studio 2022 v143 toolset will be used.  
+{{< image-width src="/images/blog/vs2022/project-manager-build-options.png" width="100%" alt="Project Manager Project Options" >}}
+{{< image-width src="/images/blog/vs2022/project-manager-building.png" width="100%" alt="Project Manager Building" >}}
+
+
+## VS2022 vs VS2019 O3DE differences
+
+The are several differences to using Visual Studio 2022 to build O3DE versus Visual Studio 2019.
```
Comment (hultonha) [inline content/blog/posts/visual-studio-2022-update.md:36]: nit: `differences to` -> `differences when`

File: content/blog/posts/visual-studio-2022-update.md [hunk 19]
```diff
@@ -0,0 +1,277 @@
...
+
+## VS2022 vs VS2019 O3DE differences
+
+The are several differences to using Visual Studio 2022 to build O3DE versus Visual Studio 2019.  
+
+### VS2022(v143) uses more memory than VS2019(v142)
+
+The most notable is that Visual Studio 2022 uses more memory when compiling O3DE than Visual Studio 2019.  
+
+{{< note >}} If the Windows OS becomes unresponsive or build times start to expand to over several hours, this is indication that build is memory limited and users should check how much RAM they have available when building.
```
Comment (hultonha) [inline content/blog/posts/visual-studio-2022-update.md:42]: update: `this is indication` -> `this is an indication`

File: content/blog/posts/visual-studio-2022-update.md [hunk 20]
```diff
@@ -0,0 +1,277 @@
...
+A good way to mitigate build memory issues with the build using Visual Studio 2022 is to adjust the compilation parameters for compiling multiple projects in parallel(`/m:<N>` or `/maxcpucount:<N>`) and multiple cl.exe processes per project(`/p:CL_MPCount=<N>`).
+{{< /note >}}  
+
+When building from the command line on a machine with 64 logical cores and 64 GiB of ram, the parameters of `/maxcpucount` and `/p:CL_MPCount` can be supplied to CMake build wrapper command(`cmake --build`) to control the number of cl.exe processes that get launched.
```
Comment (hultonha) [inline content/blog/posts/visual-studio-2022-update.md:46]: nit: `ram` -> `RAM`

File: content/blog/posts/visual-studio-2022-update.md [hunk 21]
```diff
@@ -0,0 +1,277 @@
...
+The following command restricts msbuild to build no more than 8 Visual Studio C++ Projects(.vcxproj) in parallel at a given time and no more than 16 `cl.exe` process per .vcxproj at once.
```
Comment (hultonha) [inline content/blog/posts/visual-studio-2022-update.md:47]: suggestion: Might be worth wrapping msbuild in backticks (for the command) or use MSBUILD for the name

File: content/blog/posts/visual-studio-2022-update.md [hunk 22]
```diff
@@ -0,0 +1,277 @@
...
+```
+cmake --build build/windows --target Editor --config profile -- /maxcpucount:8 /p:CL_MPCount=16
+```
+
+The equivalent options in the Visual Studio 2022 IDE can be set via following IDE options
```
Comment (hultonha) [inline content/blog/posts/visual-studio-2022-update.md:52]: nit: `via following` -> `via the following`

File: content/blog/posts/visual-studio-2022-update.md [hunk 23]
```diff
@@ -0,0 +1,277 @@
...
+* Setting the maximum number of parallel project builds value in the `Tools -> Options -> Projects and Solutions -> Build and Run` menu.  
+{{< image-width src="/images/blog/vs2022/vs2022-parallel-project-builds-8.png" width="100%" alt="Build and Run" >}}
+* Setting the maximum concurrent C++ compilations option in the `Tools -> Options -> Projects and Solutions -> VC++ Project Settings` menu.  
+{{< image-width src="/images/blog/vs2022/vs2022-concurrent-compiliations-16.png" width="100%" alt="VC++ Project Settings" >}}
+
+This can to help alliviate any memory pressure that is caused by building the O3DE.
```
Comment (hultonha) [inline content/blog/posts/visual-studio-2022-update.md:58]: question: May want to change `the O3DE` to just `O3DE` here

File: content/blog/posts/visual-studio-2022-update.md [hunk 24]
```diff
@@ -0,0 +1,277 @@
...
+```powershell
+PS C:\code\o3de> cmake -B build/windows_vs143_cli -S .
+-- Building for: Visual Studio 17 2022
+-- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.19044.
+-- The C compiler identification is MSVC 19.33.31630.0
+-- The CXX compiler identification is MSVC 19.33.31630.0
+```
+
+### Using O3DE CMake Presets (CMake 3.23+)
+O3DE provides a [CMakePresets.json](https://github.com/o3de/o3de/blob/main/CMakePresets.json) in the repo root, which leverages the [CMakePresets](https://cmake.org/cmake/help/v3.23/manual/cmake-presets.7.html) feature which is way to shared configuration, build and test settings with other users.
```
Comment (hultonha) [inline content/blog/posts/visual-studio-2022-update.md:112]: update: `feature which is way to shared configuration` -> `feature which is a way to share configuration`

**Review by hultonha — APPROVED** [review]
This post looks excellent @lumberyard-employee-dm! Super helpful to have all this information in one place and all the screen shots are incredibly helpful too. I just have a bunch of minor nits but overall the blogpost is really excellent. Awesome work! 😄

File: content/blog/posts/visual-studio-2022-update.md (hunk 10 above)
Comment (lumberyard-employee-dm) [reply content/blog/posts/visual-studio-2022-update.md:9]: I'll let the grammatical experts determine which one to use.

File: content/blog/posts/visual-studio-2022-update.md (hunk 11 above)
Comment (lumberyard-employee-dm) [reply content/blog/posts/visual-studio-2022-update.md:16]: There is some irony here.

We can't figure out where it should be called "the O3DE" or "O3DE", but we are fairly certain it should be called "Visual Studio 2022" and not "Visual Studio 2022"

Is it the "The Batman" or just "Batman"?

File: content/blog/posts/visual-studio-2022-update.md (hunk 21 above)
Comment (lumberyard-employee-dm) [reply content/blog/posts/visual-studio-2022-update.md:47]: I'll specify it using backticks

File: content/blog/posts/visual-studio-2022-update.md [hunk 25]
```diff
@@ -0,0 +1,277 @@
...
+
+This can to help alliviate any memory pressure that is caused by building the O3DE.
+
+### VS2022(v143) is stricter at triggering warnings than VS2019(v142)
+The second notable differences is that Visual Studio 2022 is better at triggering compiler warnings than VS2019.  
+So occasionally code that would compile successfully in VS2019, may fail with a compiler warning in VS2022.  
+The triggered warnings are generally around [unsigned/signed](https://learn.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-3-c4018?view=msvc-170) comparison mismatches and [usused variables](https://learn.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-4-c4189?view=msvc-170).
+
+
+
```
[the remark this answers is not in the document]
Comment (lumberyard-employee-dm) [reply content/blog/posts/visual-studio-2022-update.md:66]: This this have any practical effect on the rendering of the webpage?

File: content/blog/posts/visual-studio-2022-update.md [hunk 26]
```diff
@@ -0,0 +1,277 @@
...
+                "type": "equals",
+                "lhs": "${hostSystemName}",
+                "rhs": "Windows"
+            }
+        }
+    ]
+}
+```
+
+I use this preset file to configure the "Atom Sample Viewer" project in a project-centric workflow against the latest version of Visual Studio by only specifying the preset name to the CMake CLI as follows.
```
[the remark this answers is not in the document]
Comment (lumberyard-employee-dm) [reply content/blog/posts/visual-studio-2022-update.md:274]: Well looking at the recommendations for submitting a blog post: https://www.o3de.org/docs/contributing/to-docs/blog-posts/#do
One of the "do" bullets points mentions putting a human face on a technical product (emphasis mine)
> Interviews with, or profiles of, important and established community members. **People put a human face on a technical product**, and the world reliably likes to hear humans say what they have to say. In this case, it should be professional observations, knowledge, and commentary—and should be specifically related to O3DE. Personal opinions, as ever, should be largely technical.

I'll rephrase it to read generally though as I like that approach better.

File: content/blog/posts/visual-studio-2022-update.md (hunk 10 above)
Comment (hultonha) [reply content/blog/posts/visual-studio-2022-update.md]: Good idea! 🙂

File: content/blog/posts/visual-studio-2022-update.md (hunk 25 above)
[the remark this answers is not in the document]
Comment (hultonha) [reply content/blog/posts/visual-studio-2022-update.md]: It does not but it keeps the source files cleaner and depending on the markdown previewer it _can_ make a difference in some cases as far as I know. It is recommended here - https://github.com/DavidAnson/markdownlint/blob/v0.26.2/doc/Rules.md#md022

File: content/blog/posts/visual-studio-2022-update.md (hunk 10 above)
Comment (chanmosq) [reply content/blog/posts/visual-studio-2022-update.md]: This is a tough one because grammatically, both work: E.g. "The United States of America" vs. "England"

So the determining factor is usually consistency. Most docs in the O3DE documentation use "O3DE", without "the". So I suggest go with that!

File: content/blog/posts/visual-studio-2022-update.md [hunk 27]
```diff
@@ -0,0 +1,301 @@
+---
+title: "O3DE Visual Studio 2022 Support!"
+date: 2022-11-09
+slug: o3de-vs2022-initial-support
+author: lumberyard-employee-dm
+blog_img: "/images/blog/announcement_thumbnail.jpg"
+full_img: ""
+---
+With the 22.10.0 release of **Open 3D Engine (O3DE)** Visual Studio 2022 (VS2022) is now fully supported within O3DE.
```
Comment (chanmosq) [inline content/blog/posts/visual-studio-2022-update.md:9]:
```suggestion
With the 22.10.0 release of **Open 3D Engine (O3DE)**, Visual Studio 2022 (VS2022) is now fully supported within O3DE.
```

File: content/blog/posts/visual-studio-2022-update.md (hunk 10 above)
Comment (lumberyard-employee-dm) [reply content/blog/posts/visual-studio-2022-update.md]: Ok, "O3DE"  without the "the" it is.

**Review by aFinchy — APPROVED** [review]
Everything on the Formatting side looks good to me. 

Once we get the other comments resolved, let me know so we can merge. The blog is set to go live Nov 9th.

**Comment by lumberyard-employee-dm** [comment]
> Everything on the Formatting side looks good to me.
> 
> Once we get the other comments resolved, let me know so we can merge. The blog is set to go live Nov 9th.

All the other comments are resolved.
It should be good to merge now.
